client/src/pages/charactersGalery/CharacterGallery.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import ReactPaginate from "https://cdn.skypack.dev/react-paginate@7.1.3";
```

Comment:
I think its good idea to separated to ui-kit structure for flexibility and maintain the change will be only in one place instead of all the application